pkg/build

```diff
@@ -29,15 +29,20 @@ MANIFESTS = \
 	pkg/ssh/manifest.json \
 	$(NULL)
 
-.PHONY: all-webpack
+.PHONY: all-webpack ls-webpack-inputs
 all-webpack: $(MANIFESTS)
 
 V_TAR = $(V_TAR_$(V))
 V_TAR_ = $(V_TAR_$(AM_DEFAULT_VERBOSITY))
 V_TAR_0 = @echo "  TAR     " $@;
 
+ls-webpack-inputs:
+	ls -1 package-lock.json $(WEBPACK_INPUTS) | uniq
+
 webpack-jumpstart.tar: $(MANIFESTS) package-lock.json pkg/build
-	$(V_TAR) tar cf webpack-jumpstart.tar dist package-lock.json
+	@pkg/build -s ls-webpack-inputs | xargs sha256sum > .sha256sums
```

Comment:
What's the advantage of doing this complicated way (sub-make and new internal phony target) as opposed to just putting the `ls` here? Also, Aren't `$WEBPACK_INPUTS` already single files? Why do they need to go through `ls` again, as opposed to `sha256sum package-lock.json $(WEBPACK_INPUTS)`?

Reply:
This part is annoying and complicated, for two reasons:
 - in the first `make` run, `WEBPACK_INPUTS` is empty, because the `Makefile.deps` weren't created yet.  The second `make` run is to give it a chance to read those files in after webpack runs
 - I use `ls` instead of `echo` because there is quite a lot of duplication in `$(WEBPACK_INPUTS)`, and I don't know a better way than `uniq` (which requires separate, sorted, lines — as `ls` produces).  Probably I could drop the `xargs` use here, though, in favour of a `$()`.  Ideas welcome!

Reply:
Amen to "annoying and complicated" :grin: -- see #16005, is there really no simpler way to do this?